boot/zephyr/boards/uart_detect.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_MCUBOOT_SERIAL=y
```

Comment:
this is not a board dedicated file, how you want to use it? with -DCONF_FILE?

Reply:
Yes, Below is a sample:
cmake -DBOARD=nrf52_moderate -DBOARD_ROOT=/home/frank/work/project/nrf52_moderate -DCONF_FILE="prj.conf boards/uart_detect.conf" ..